src/nwb_benchmarks/benchmarks/network_tracking_remote_file_reading.py

```diff
@@ -7,10 +7,14 @@
     get_s3_url,
     network_activity_tracker,
     read_hdf5_fsspec_no_cache,
+    read_hdf5_fsspec_with_cache,
     read_hdf5_nwbfile_fsspec_no_cache,
+    read_hdf5_nwbfile_fsspec_with_cache,
     read_hdf5_nwbfile_remfile,
+    read_hdf5_nwbfile_remfile_with_cache,
     read_hdf5_nwbfile_ros3,
     read_hdf5_remfile,
+    read_hdf5_remfile_with_cache,
     read_hdf5_ros3,
 )
```

Comment:
@sinha-r for the imports of the new functions to work `from nwb_benchmarks.core import read_hdf5_fsspec_with_cache` we also need to add them to the public interface of `nwb_benchmarks.core` by adding them to the `__all__` list in the `__init__.py` file. I'll submit a fix shortly, but just wanted to let you know for future reference. 

https://github.com/NeurodataWithoutBorders/nwb_benchmarks/blob/e9eeab25aea9a29cb7592d51ef722abd5f6519be/src/nwb_benchmarks/core/__init__.py#L19-L33

Reply:
Fixed in [d2b4777](https://github.com/NeurodataWithoutBorders/nwb_benchmarks/pull/43/commits/d2b477700fff519f744e02df95dc9c72ad85cada)